packages/types/src/Moment.ts

```diff
@@ -7,16 +7,17 @@ import bnToBn from '@polkadot/util/bn/toBn';
 import bnToU8a from '@polkadot/util/bn/toU8a';
 import isU8a from '@polkadot/util/is/u8a';
 import u8aToBn from '@polkadot/util/u8a/toBn';
+import u8aToHex from '@polkadot/util/u8a/toHex';
 
-import Base from './codec/Base';
+import { Codec } from './types';
 
 const BITLENGTH = 64;
 
 // A wrapper around seconds/timestamps. Internally the representation only has
 // second precicion (aligning with Rust), so any numbers passed an/out are always
 // per-second. For any encoding/decoding the 1000 multiplier would be applied to
 // get it in line with JavaScript formats
-export default class Moment extends Base<Date> {
+export default class Moment extends Date implements Codec<Moment> {
```

Comment:
Is this correct, i.e. `Codec<Moment>` (e.g. Bool above extends built-in JS `Boolean` class, one of the 2 is wrong)